app/screens/thread/thread.ios.test.js

```diff
@@ -12,10 +12,23 @@ import {shallowWithIntl} from 'test/intl-test-helper';
 
 import ThreadIOS from './thread.ios';
 
+jest.mock('react-native-navigation', () => {
+    return {
+        Navigation: {
+            events: () => ({
+                registerAppLaunchedListener: jest.fn(),
+            }),
+            registerComponent: jest.fn(),
+            setLazyComponentRegistrator: jest.fn(),
+        },
+    };
+});
+
```

Comment:
can you move this to https://github.com/mattermost/mattermost-mobile/blob/master/test/setup.js#L213